lms/djangoapps/learner_dashboard/views.py

```diff
@@ -0,0 +1,52 @@
+"""
+Handles requests for views, returning page frame
+"""
+
+import logging
+from urlparse import urljoin
+
+from django.conf import settings
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.http import Http404
+
+from edxmako.shortcuts import render_to_response
+from openedx.core.djangoapps.programs.utils import get_programs_for_dashboard
+from learner_dashboard.models import LearnerDashboardConfiguration
+from student.views import get_course_enrollments
+
+
+log = logging.getLogger(__name__)
+
+
+@login_required
+@ensure_csrf_cookie
+def view_programs(requests):
+    if not LearnerDashboardConfiguration.current().enable_learner_dashboard:
```

Comment:
What other cases are you thinking about? I don't know what action I can take from your question, because my answer to your question above would be a hesitant "yes"